open-api/rest-catalog-open-api.yaml

```diff
@@ -100,6 +121,16 @@ paths:
 
         Common catalog configuration settings are documented at
         https://iceberg.apache.org/docs/latest/configuration/#catalog-properties
+
+
+        Capabilities are used by the server to signal supported functionality and can be:
+        - tables (default capability in case the `capabilities` property doesn't exist or is empty in the response)
+        - views
+        - remote-signing
+        - multi-table-commit
+        - register-table
+        - table-metrics
+        - oauth2
```

Comment:
What does `oauth2` mean exactly? Is it that only OAuth2 or anonymous are supported by this spec? And if so, which OAuth2 flows are supported? How does SigV4 fit here?

Reply:
`oauth2` refers to the `/tokens` endpoint as it currently exists. See also https://github.com/apache/iceberg/pull/9940/files#r1645193788

Reply:
In this context `oauth2` may be misleading (as discussed in depth on the `dev` mail list). Would it be possible to rename this value to `local-tokens` (other suggestions welcome)? My point is to avoid confusion with [RFC6749](https://www.rfc-editor.org/rfc/rfc6749) since the behaviour of the `/tokens` endpoint is quite specific to the Iceberg REST spec, and IIRC there was some consensus for deprecating the `/tokens` endpoint in the REST spec.